lib/src/models/jobs.dart

```diff
@@ -0,0 +1,230 @@
+import 'package:json_annotation/json_annotation.dart';
+import 'package:meta/meta.dart';
+
+part 'jobs.g.dart';
+
+@JsonEnum(fieldRename: FieldRename.screamingSnake)
+enum JobStatus {
+  queued,
+  started,
+  running,
+  failed,
+  success,
+  cancelled,
+  cancellationRequested,
+}
+
+@immutable
+@JsonSerializable(fieldRename: FieldRename.snake)
```

Comment:
Could you please add includeIfNull: false to the JsonSerializable annotation? We encountered issues with a change in the Mistral API where fields with null values were being sent, and they were rejected because they should not have been included.

Just in case some new optional params will be added.

Reply:
Sure 😁  Should I add it only in the 2 places you mentioned in the review or should they be added to each `@JsonSerializable' annotation? 🤔